src/media/mod.rs

```diff
@@ -141,13 +141,22 @@ impl FromJson for ProgressInfo {
 /// [`DraftTweet::media_ids`]: ../tweet/struct.DraftTweet.html#method.media_ids
 #[derive(Copy, Clone, Debug)]
 pub struct MediaHandle {
-    /// The numeric ID that can be used to reference the media.
+    /// The ID that can be used as the media's reference.
```

Comment:
"as the media's reference" sounds weird to me. It's not a phrase i see all that often so i'd rather not use it here.

Reply:
ok